tests/tools/Makefile

```diff
@@ -29,6 +30,6 @@ $(BUILDDIR)/git-validation:
 $(BUILDDIR)/go-md2man:
 	$(call go-build,./vendor/github.com/cpuguy83/go-md2man)
 
-$(BUILDDIR)/golangci-lint: VERSION=v1.55.2
 $(BUILDDIR)/golangci-lint:
-	curl -fsSL https://raw.githubusercontent.com/golangci/golangci-lint/$(VERSION)/install.sh | sh -s -- -b ./$(BUILDDIR) $(VERSION)
+$(BUILDDIR)/golangci-lint:
```

Comment:
Does it make sense to have this line twice?

Reply:
No